src/app/containers/ArticleFigure/fixtureData.jsx

```diff
@@ -166,6 +166,7 @@ const generateFixtureData = ({
         width={width}
         type={type}
         lazyLoad={lazyLoad}
+        service={serviceContextStubNews.service}
```

Comment:
This value would have to be dynamic, to allow use switch `Caption` font based on service.